openllm-core/src/openllm_core/config/configuration_baichuan.py

```diff
@@ -3,7 +3,8 @@
 
 import openllm_core
 
-from openllm_core._prompt import process_prompt
+from openllm_core.prompt import PromptTemplate
+from openllm_core.prompt import process_prompt
```

Comment:
Actually, since you already export `PromptTemplate` and `process_prompt` in `__init__.py`, no need to explicit import it here, can just use `openllm_core.PromptTemplate` and `openllm_core.process_prompt`

Reply:
As far as I know, having 'overlapping' imports shouldn't affect module size or runtime. imo it might be better code legibility to just use `PromptTemplate` rather than `openllm_core.PromptTemplate`. But I could make the change if you disagree.

Reply:
it actually affects import time because you are accessing the module, whereas openllm_core is loaded lazily in the perspective of `openllm`.

THe opt is minute, probably doesn't matter that much.

My suggestion only applies since we also export these class and function in `openllm_core`, and we already import `openllm_core`, so there is no need for explicit import.

At the end of the day, it is a style choice (doesn't matter that much)

Reply:
Sounds good. Valid point but if you don't mind I will leave it as is since I think it provides better legibility. Marking as resolved 😅